app/src/interfaces/group-raw/group-raw.vue

```diff
@@ -19,65 +19,30 @@
 	</div>
 </template>
 
-<script lang="ts">
+<script setup lang="ts">
 import { Field, ValidationError } from '@directus/types';
-import { defineComponent, PropType } from 'vue';
-export default defineComponent({
-	name: 'InterfaceGroupRaw',
-	props: {
-		field: {
-			type: Object as PropType<Field>,
-			required: true,
-		},
-		fields: {
-			type: Array as PropType<Field[]>,
-			required: true,
-		},
-		values: {
-			type: Object as PropType<Record<string, unknown>>,
-			required: true,
-		},
-		initialValues: {
-			type: Object as PropType<Record<string, unknown>>,
-			required: true,
-		},
-		disabled: {
-			type: Boolean,
-			default: false,
-		},
-		batchMode: {
-			type: Boolean,
-			default: false,
-		},
-		batchActiveFields: {
-			type: Array as PropType<string[]>,
-			default: () => [],
-		},
-		primaryKey: {
-			type: [Number, String],
-			required: true,
-		},
-		loading: {
-			type: Boolean,
-			default: false,
-		},
-		validationErrors: {
-			type: Array as PropType<ValidationError[]>,
-			default: () => [],
-		},
-		badge: {
-			type: String,
-			default: null,
-		},
-		rawEditorEnabled: {
-			type: Boolean,
-			default: false,
-		},
-		direction: {
-			type: String,
-			default: undefined,
-		},
-	},
-	emits: ['apply'],
-});
+
+withDefaults(
+	defineProps<{
+		field: Field;
+		fields: Field[];
+		values: Record<string, unknown>;
+		initialValues: Record<string, unknown>;
+		primaryKey: number | string;
+		disabled?: boolean;
+		batchMode?: boolean;
+		batchActiveFields?: string[];
+		loading?: boolean;
+		validationErrors?: ValidationError[];
+		badge?: string;
+		rawEditorEnabled?: boolean;
+		direction?: string;
+	}>(),
+	{
+		batchActiveFields: () => [],
+		validationErrors: () => [],
+	}
+);
+
+defineEmits(['apply']);
```

Comment:
Same as in https://github.com/directus/directus/pull/18411/files?diff=split&w=1#r1183472789.